src/sizzle.js

```diff
@@ -150,6 +150,24 @@ var i,
 				String.fromCharCode( high >> 10 | 0xD800, high & 0x3FF | 0xDC00 );
 	},
 
+	// CSS string/identifier serialization
+	// https://drafts.csswg.org/cssom/#common-serializing-idioms
+	rcssescape = /([\0-\x1f\x7f]|^-?\d)|^-$|[^\x80-\uFFFF\w-]/g,
+	fcssescape = function( ch, numeric ) {
+
+		// We allow NULL input, unlike the draft spec as of 2015-11-09
+		// Our syntax is already broader elsewhere
```

Comment:
Any thoughts on this, @jquery/core? To be honest, I'm contemplating filing an issue against the CSSOM draft, because `\0 ` is a valid escape sequence (http://www.w3.org/TR/css-syntax/#escaping , http://www.w3.org/TR/css-syntax/#consume-an-escaped-code-point0 ), although it is [replaced with `\fffd `](http://www.w3.org/TR/css-syntax/#changes-css21):
> Any hexadecimal escape sequence such as \0 that evaluates to zero produce U+FFFD REPLACEMENT CHARACTER rather than U+0000 NULL.